scania/corporate-ui-site
# Feature/cards

**Describe pull-request**  
Add cards styling

**How to test**  
Included in this PR :

- Cards styling
- Check on hover state
- Figma: UI Kit > Cards

**Screenshots**  
![image](https://user-images.githubusercontent.com/1199101/101733461-38244e00-3abf-11eb-9ac7-752f7f54246a.png)


**Additional context**  
Merge after #121
